### PR TITLE
Feature/fallback slack channel

### PR DESCRIPTION
**Pull request by derekvanassche**

### CODE REVIEW

File: assets/common/common.go [hunk 1]
```diff
@@ -51,32 +54,35 @@ func HandleFatalError(err error, msg string) {
...
 	fmt.Fprintln(os.Stderr, fmt.Sprintf("%s: %s", msg, err.Error()))
 }
 
-func ValidateAndBuildPostBody(input ConcourseInput) (method string, values url.Values, err error, emtpy bool) {
+// ValidateAndBuildPostBody ... Directs requests to appropriate delegates depending on if we deal with a file upload or a chat message
+func ValidateAndBuildPostBody(input ConcourseInput) (method string, values url.Values, emtpy bool, err error) {
 	var exists = false
 	if method, exists = slackMethodPaths[input.Source.Method]; exists == false {
-		return "", nil, errors.New(fmt.Sprintf("Method '%s' does not exist", input.Source.Method)), false
+		return "", nil, false, fmt.Errorf("Method '%s' does not exist", input.Source.Method)
```
Comment (jaibhavaya) [inline assets/common/common.go:69]: curious, why the change in return order?

**Review by jdimmerman — APPROVED** [review]

File: assets/common/common.go (hunk 1 above)
Comment (derekvanassche) [reply assets/common/common.go:69]: It was suggested by `golint` to specify the error as the last return parameter.

**Review by jaibhavaya — APPROVED** [review]
